scripts/ci-entrypoint.sh

```diff
@@ -181,6 +181,9 @@ timeout --foreground 1800 bash -c wait_for_nodes
 
 # install cloud-provider-azure components, if using out-of-tree
 if [[ -n "${TEST_CCM:-}" ]]; then
+    if [[ -n "${TEST_WINDOWS:-}" ]]; then
```

Comment:
since this is also relevant when testing Linux only, does it make sense to remove this and always check for ready calico pods?

Reply:
"app: calico" is (unintuitively) only the windows label (shrug)

There are other labels for other parts of the current calico implementation

Reply:
let's add a code comment about that, unintuitive indeed.

Reply:
Updated. Thanks for the review.